README.md

```diff
@@ -0,0 +1,55 @@
+<p align="center"><a href="https://minecraftdev.org/"><img src="https://minecraftdev.org/assets/icon.svg" height="120" alt="logo"/></a></p>
+
+Minecraft Development for IntelliJ - Templates
+==============================================
+
+## Info and Documentation
+<a href="https://discord.gg/j6UNcfr"><img src="https://i.imgur.com/JXu9C1G.png" height="48px"></img></a>
+
+This repository is part of the Minecraft Development for IntelliJ project.
+
+This contains the default templates for this plugin.
+
+For more informations, please visit [https://mcdev.io](https://mcdev.io) for more informations about it.
+
+## Usage
+You simply need to install the plugin Minecraft Development for IntelliJ on your instance of Jetbrains IntelliJ Idea.
+
+For more informations about this plugin's installation, please see [this explanation](https://github.com/minecraft-dev/MinecraftDev?tab=readme-ov-file#installation).
+
+## Testing
+If you're in the will of modifying this templates or create your owns, please see [https://mcdev.io/docs/creating-creator-templates/](https://mcdev.io/docs/creating-creator-templates/) for more help about this.
+
+You can either choose to host your modifications on a GitHub repository or on your local machine. If you choose to host it on your own machine, please select "local" instead of "remote" (as explained in the Documentation).
+
+Developers
+----------
+
+- Project Owner - [**@DenWav** - Kyle Wood](https://github.com/DenWav)
+- [**@Minecrell**](https://github.com/Minecrell)
+- [**@PaleoCrafter** - Marvin Rösch](https://github.com/PaleoCrafter)
+- [**@RedNesto**](https://github.com/RedNesto)
+- [**@Earthcomputer** - Joseph Burton](https://github.com/Earthcomputer)
+
+<!-- #### **Significant Contributors**
+
+You can either add main project contributors here or place contributors of minecraft-dev/templates here.
```

Comment:
No need to copy this part of the readme here, will probably be removing this section in the mcdev readme anyways.